bundle/edu.gemini.itc.shared/src/main/scala/edu/gemini/itc/shared/Parameters.scala

```diff
@@ -108,84 +108,79 @@ sealed trait CalculationMethod {
   require(isSpectroscopy != isImaging)
```

Comment:
Random observation unrelated to changes: maybe we could just define `final def isImaging: Boolean = !isSpectroscopy` and not `require` anything.

Reply:
right, good idea